web/src/test/java/com/replica/StreetEdgeExporterTest.java

```diff
@@ -32,22 +29,42 @@ public void testExportEndToEnd() throws IOException {
         File expectedOutputLocation = new File(EXPORT_FILES_DIR + "street_edges.csv");
         CSVParser parser = CSVParser.parse(expectedOutputLocation, StandardCharsets.UTF_8, format);
         List<CSVRecord> records = parser.getRecords();
-        assertEquals(1095756, records.size());
+        assertEquals(1147753, records.size());
```

Comment:
any idea why we get a larger number of edges now? is this due to the dupes mentioned below?

Reply:
Great question! We've seen this in every iteration of the core update that we've done, and it's never super clear what the differences are, but usually it's because of subtle changes in road types that graphhopper does/doesn't accept during OSM processing (for ex small service/private roads). The newer graphhopper versions seem to be a bit more accepting of these minor road types.

Probably a good thing to look into more in the future, as we've had ongoing discussions for a long time about paring down the roads we represent in the street export. But for now, when I do spot-checks of major roadways, I see essentially identical links, which is a good sign to me